### DISCUSSION TITLE
BelongsToMany Relationship possible?

### DISCUSSION THREAD

**Discussion by SebastianUhlig** [awcodes/filament-curator#164] [Q&A]
Didn't find anything in any issues or QAs: 
I only read that multiple images are only possible via the builder function. But is there a way to assign multiple images to a model via a belongsToMany relationship?
I tried to create a BelongsToMany relation in the model and used the relationship name as name in the CuratorPicker form field, but it doesn't seem to work. Has it changed?

Form Field
`CuratorPicker::make('images')->label(__('Select'))`

Model
```
public function images(): BelongsToMany
{
    return $this->belongsToMany(Media::class, 'product_media');
}
```

Table
```
$table->foreignId('product_id')->constrained()->cascadeOnDelete();
$table->foreignId('media_id')->constrained()->cascadeOnDelete();
```

(sorry if I duplicate the topic. Haven't been able to find anything on this specific topic since v2)

**Comment by awcodes** [comment]
The CuratorPicker, as of right now doesn't support multiple images. You could use a Repeater with CuratorPicker in its schema to achieve what you are trying to do.
